tooling/packherd/test/packherd.spec.ts

```diff
@@ -0,0 +1,92 @@
+import {
+  CreateBundle,
+  CreateBundleOpts,
+  CreateBundleResult,
+  CreateBundleOutputFile,
+  packherd,
+} from '../src/packherd'
+import { expect } from 'chai'
+import spok from 'spok'
```

Comment:
Any thoughts on not introducing a `spok` dependency here? The assertions here might have more boilerplate without it, but there's not that many of them.

Reply:
Updated here: [`c3e4634` (#23257)](https://github.com/cypress-io/cypress/pull/23257/commits/c3e46342d4b31d7bf8f5b89ccd5c512ac4e53a19)

(though it's going to be harder on subsequent PRs 😄)